extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -276,17 +279,17 @@ export class SchemaCompareDialog {
 			this.formBuilder.removeFormItem(this.sourceNoActiveConnectionsText);
 			this.formBuilder.removeFormItem(this.sourceServerComponent);
 			this.formBuilder.removeFormItem(this.sourceDatabaseComponent);
-			this.formBuilder.insertFormItem(this.sourceDacpacComponent, 1, { horizontal: true });
+			this.formBuilder.insertFormItem(this.sourceDacpacComponent, 2, { horizontal: true, titleFontSize: titleFontSize });
```

Comment:
why did we increase the index?

Reply:
The ordering of the source dialog components since the indexes changed after formComponentGroups were added in this PR #5329. Had to increase the indexes by one because the "Source" title was added. Before this index change, going back and forth between the buttons made the order wrong and look like this: 
![image](https://user-images.githubusercontent.com/31145923/57169138-ec290780-6db9-11e9-8635-e12e49d9de09.png)